cfme/tests/automate/test_file_import.py

```diff
@@ -31,3 +37,56 @@ def test_domain_import_file(import_datastore, import_data):
             3. Import should work. Check imported or not.
     """
     assert import_datastore.exists
+
+
+@pytest.mark.tier(2)
+@pytest.mark.meta(automates=[1720611])
+@pytest.mark.parametrize("files", ["datastore_blank.zip", "dialog_blank.yml"],
+                         ids=["datastore", "dialog"])
+@pytest.mark.uncollectif(lambda files: files == "dialog_blank.yml" and
+                         BZ(1720611, forced_streams=['5.10']).blocks)
+def test_upload_blank_file(appliance, files):
+    """
+    Bugzilla:
+        1720611
+
+    Polarion:
+        assignee: ghubale
+        initialEstimate: 1/8h
+        caseposneg: negative
+        startsin: 5.10
+        casecomponent: Automate
+        testSteps:
+            1. Create blank zip(test.zip) and yaml(test.yml) file
+            2. Navigate to Automation > Automate > Import/Export and upload test.zip file
+            3. Navigate to Automation > Automate > Customization > Import/Export and upload test.yml
+        expectedResults:
+            1.
+            2. Error message should be displayed
+            3. Error message should be displayed
+    """
+    # Download datastore file from FTP server
+    fs = FTPClientWrapper(cfme_data.ftpserver.entities.datastores)
+    file_path = fs.download(files)
+
+    if files == "dialog_blank.yml":
+        with LogValidator("/var/www/miq/vmdb/log/production.log",
+                          failure_patterns=[".*FATAL.*"]).waiting(timeout=120):
+
+            # Import dialog yml to appliance
+            import_export = DialogImportExport(appliance)
+            view = navigate_to(import_export, "DialogImportExport")
+            view.upload_file.fill(file_path)
+            view.upload.click()
+            view.flash.assert_message('Error: the uploaded file is blank')
+    else:
+        # Import datastore file to appliance
+        datastore = appliance.collections.automate_import_exports.instantiate(
+            import_type="file", file_path=file_path
+        )
+        view = navigate_to(datastore.parent, "All")
```

Comment:
_Optional_ consider navigating via `appliance.collections.automate_import_exports` instead, it is a bit more explicit.

Reply:
Sounds Good!